detect/decoder.go

```diff
@@ -0,0 +1,256 @@
+package detect
+
+import (
+	"bytes"
+	"encoding/base64"
+	"regexp"
+	"unicode"
+
+	"github.com/rs/zerolog/log"
+)
+
+var b64LikelyChars [128]byte
+var b64Regexp = regexp.MustCompile(`[\w\/\-\+]{16,}={0,3}`)
```

Comment:
**Question**: Where did 16 come from?

Reply:
It's an arbitrary minimum length to filter out low-value secrets, similar to entropy. 16 because base64 segments are multiples of 4.

Reply:
And also the average password length is around 9-11 characters based on a google search, after b64 encoding things in that range you end up with about 16 as a decent min.